core/src/main/java/dk/alexandra/fresco/framework/util/AesCtrDrbgFactory.java

```diff
@@ -0,0 +1,74 @@
+package dk.alexandra.fresco.framework.util;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+/**
+ * Factory class for {@link AesCtrDrbg} providing various ways to seed the DRBG.
+ */
+public class AesCtrDrbgFactory {
+
+  /**
+   * The hash used to for derived seeds.
+   */
+  public static final String HASH_ALGORITHM = "SHA-256";
+
+  private AesCtrDrbgFactory() {
+    // Should not be instantiated
+  }
+
+  /**
+   * Creates a new DRBG with a seed derived from a given sequence of bytes (of any length).
+   *
+   * <p>
+   * Internally this this essentially works by computing the seed as
+   * <code> seed = hash(bytes)</code>
+   * where <code>hash</code> is the hash function {@value #HASH_ALGORITHM}.
+   * </p>
+   * <p>
+   * Note: This method should be used with caution as, depending on the context, a seed derived
+   * from a low entropy sequence of bytes can be insecure to use.
+   * </p>
+   *
+   * @param bytes a sequence of bytes from which to derive the seed
+   * @return a new DRBG
+   * @throws NoSuchAlgorithmException if the hash algorithm {@value #HASH_ALGORITHM} is not
+   *      available on the system
+   */
+  public static Drbg fromDerivedSeed(byte... bytes) throws NoSuchAlgorithmException {
```

Comment:
It is annoying that this method is typed with an exception that cannot be controlled by the caller. 

I suggest you wrap it with ExceptionConverter.safe so callers do not have to handle a broken jvm

Reply:
Your right, I will change this.